ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2954,6 +2974,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-long-duration
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones
+- as: vsphere-upi-baselinecaps-v414-p2-f14
```

Comment:
How about turn one existing v411 job to v414 instead of adding a new one.

Reply:
'v4.11' is still the valid value for baselineCapablitySet in 4.14 version.

If replacing v4.11, we only test the latest 3 capability set v4.x + None in CI in the future?

Reply:
If I remember we discussed such situations before, seem like we did not have some v4.x capability support policy on 4.N version yet, my concern is we come to 4.20, do we still need to v4.11 capability suite?

Reply:
> my concern is if we come to 4.20 OCP version, do we still need to v4.11 capability suite?

I'm not sure. But even if it will be used, I guess the probability might be very low? Anyway, we have test case to cover such scenario, so might be safe to remove older version from CI. 
Let me update.

Reply:
I updated as below:
aws-ipi-baselinecaps-v413 -> aws-ipi-baselinecaps-v414
gcp-upi-baselinecaps-v411 -> gcp-upi-baselinecaps-v413

so that we could cover the scenario UPI without enabling MachineAPI.

@jianlinliu please help to review again, thanks.